### DISCUSSION TITLE
Make it possible to use non-inertia same-tab links for MenuItem instances

### DISCUSSION THREAD

**Discussion by piljac1** [laravel/nova-issues#4676] [Ideas & Feature Requests] [answered]
Hello! With Nova 4.7.0, we were using the `path` method to define a `MenuItem` link (that is not a Nova link, but is an internal link which makes a redirection to an external link) and it worked flawlessly.

```php
MenuItem::make('Logout')
    ->path('/logout')
```

Then we updated to Nova 4.11.1, and it stopped working giving us the following CORS error:

> Access to XMLHttpRequest at 'https://some-external-auth0-link.com/v2/logout?returnTo=https%3A%2F%2Four-site.test%2Fadmin&client_id=X4mSCT8twvYXzzi7QE8zYKRJoXatzNft' (redirected from 'https://our-site.test/admin/logout') from origin 'https://our-site.test/' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

> Request URL: https://some-external-auth0-link.com/v2/logout?returnTo=https%3A%2F%2Four-site.test%2Fadmin&client_id=X4mSCT8twvYXzzi7QE8zYKRJoXatzNft
Referrer Policy: strict-origin-when-cross-origin

Our solution was to use the `external` method for the component to use an `a` tag instead of the Inertia `Link` component. For this Nova version, `external` was opening the link in the same window and that's the behavior we wanted. Little side note, in the Nova 4.11.1 changelogs it says that external links should now open in a new tab, but that's not the case.

Then, we updated to the latest Nova version as of now (4.12.14) and now external links really do open in a new tab (which is to be expected due to the name). However, this leaves us with undesirable options:
1. We keep using 4.11.1 and don't update
2. We update but we use an external link (target blank) for our logout (which is terrible UX)
3. We do our own custom class/component which extends the `MenuItem` class/component and add a method to use a simple non-blank `a` tag

So, that being said, is it possible to offer us a way to use a plain `a` tag which is not an Inertia `Link` or a `_blank` link?

Thanks in advance!

**Comment by crynobone** [comment, marked as answer]
You can now specify `openOnNewTab()` to open external link on a separate tab. Otherwise, it will default to open on the current page starting from `4.13.0`:

```php
MenuItem::externalLink('Documentation', 'https://nova.laravel.com/docs')->openOnNewTab();
```
